src/components/AppLayout/TopNav/TopNavWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box } from "@saleor/macaw-ui/next";
+import React from "react";
+
+import { topBarHeight } from "../consts";
+
+export const TopNavWrapper: React.FC = ({ children }) => (
+  <Box
+    display="flex"
+    alignItems="center"
+    paddingX={9}
+    paddingY={8}
+    borderBottomWidth={1}
+    borderBottomStyle="solid"
+    borderColor="neutralPlain"
+    position="relative"
+    data-test-id="page-header"
+    gap={7}
+    __height={topBarHeight}
+    __gridArea="nav"
```

Comment:
shouldnt it be referenced in Macaw and marked as "todo" to change it?

Reply:
We have https://github.com/saleor/saleor-dashboard/pull/3213 that will remove those areas